metaphor/bigquery/config.py

```diff
@@ -62,6 +64,9 @@ class BigQueryQueryLogConfig:
         default_factory=lambda: ProcessQueryConfig()
     )
 
+    # Maximum allowed requests per minute to the log entries API
+    rate_limit: int = DEFAULT_RATE_LIMIT
```

Comment:
Maybe just call this `max_requests_per_minute` so it's easier to understand.
```suggestion
    max_requests_per_minute: int = DEFAULT_MAX_REQUESTS_PER_MINUTE
```